src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
Don't we need this for python files that we execute directly, like this one?

Reply:
Do we actually _execute_ this one?  I can't see how we would given it's permissions are 644 -- so no execute bit set on it and I don't see the RPM packaging adding the execute permission on packaging it up.

Reply:
It seems we do execute it and the perms in my local repo got changed.  Not sure how.  In any case, I have reverted this one.